src/eredis.erl

```diff
@@ -64,7 +64,12 @@ start_link() ->
 %% such as `{local, atom()}'; for all options see `ServerName' at
 %% <a href="https://erlang.org/doc/man/gen_server.html#start_link-4">gen_server:start_link/4</a>;
 %% default: no name</dd>
-%% </dl>
+%% <dt>`{sentinel, SentinelOptions}'</dt><dd> Enables client for sentinel mode; options are required
+%% to connect to sentinel cluster; default: undefined </dd>
+%% <dd> Sentinel Options: `{master_group, master_group_name}' - Atom, default: mymaster;
+%% `{addresses, [{Host, Port}]}' - List of {Host, Port} tuples, default: [{"127.0.0.1", 26379}];
```

Comment:
I'm thinking about the name `addresses` here. An address is an IP address, but here we have a list of `{Host, Port}` tuples. A name like hostportpairs is not very nice though... What do you think about calling it `endpoints`?

(We will not be able to change this in the future. That's why I want to be extra careful with the API including option names.)